clusters-rke1/batch_scale.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+KUBE_CONFIG=${1}
+BATCH_NUM_NODES=${2:-50}
+TARGET_NUM_DOWNSTREAMS=${3:-150}
+
+export KUBECONFIG="${KUBE_CONFIG}"
+
+function get_heap_logs() {
+    for pod in $(kubectl -n cattle-system get pods --no-headers -l app=rancher | cut -d ' ' -f1); do
+        echo getting profile for "${pod}"
+        kubectl -n cattle-system exec "${pod}" -- curl -s http://localhost:6060/debug/pprof/heap -o profile.log
```

Comment:
Is port `6060` a standard meaning does it change between cluster types?

Reply:
6060 is specific to whatever http server port the application is using to ListenAndServe with, [pprof](https://pkg.go.dev/net/http/pprof) can then profile on that port. In this case [Rancher](https://github.com/rancher/rancher/blob/127d5b63a7ce1ca26aea9266926997f60bfad4f4/main.go#L25) is using 6060.